qiskit_ibm_runtime/session.py

```diff
@@ -193,6 +193,50 @@ def backend(self) -> Optional[str]:
         """
         return self._backend
 
+    def status(self) -> Optional[str]:
+        """Return current session status.
+
+        Returns:
+            A string describing the current status of the session.
+        """
+        details = self.details()
+        if details:
+            state = details["state"]
+            accepting_jobs = details["accepting_jobs"]
+            if state == "open":
+                return "Pending"
+            if state == "active" and accepting_jobs:
+                return "In progress, accepting new jobs"
+            if state == "active" and not accepting_jobs:
+                return "In progress, not accepting new jobs"
+            return state.capitalize()
+
```

Comment:
@jyu00 is something like this what you had in mind?

Reply:
I think `inactive` should also be mapped to `Pending`. 

In addition, the docstring should list out the possible statuses and explain what they mean, including `None`.